buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -398,7 +341,10 @@
     },
     "CrossgenArg": {
       "value": ""
-    }
+    },
+    "AgentToolsPath": {
+      "value": "c:\\tools\\AgentTools\\"
```

Comment:
We should make sure that if we set this an environment variable it overrides this path. It is find to default it but we need to make sure we don't hard-code a path we cannot override on the agent. If we cannot override this then I'm going to suggest we don't define it in the definitions.

Reply:
Looking at the pre-defined properties (https://docs.microsoft.com/en-us/vsts/build-release/concepts/definitions/build/variables?tabs=batch) perhaps we should use `Agent.HomeDirectory`. So default this to `$(Agent.HomeDirectory)\\AgentTools`.